content/07.pathogenesis.md

```diff
@@ -372,32 +364,33 @@ In contrast, a model using the prevalence of antibodies among residents of Wuhan
+However, in locations without universal testing protocols, it is impossible to identify all infected individuals because so many asymptomatic or mild cases go undetected.
 Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
-Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854], and this estimate was also supported by a repeated cross-sectional serosurvey conducted in New York City that revealed an estimated IFR of 0.97% [@doi:10.1038/s41586-020-2912-6].
+Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780].
+This was also supported by a repeated cross-sectional serosurvey conducted in New York City that estimated the IFR as 0.97% [@doi:10.1038/s41586-020-2912-6].
 All of these estimates note that IFR varies widely around the world.
 Estimates of infection rates are becoming more feasible as more data becomes available for modeling and will be bolstered as serological testing becomes more common and available.
 
 ### Dynamics of Transmission
 
 Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number.
-Thus, accurate estimates are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
+Accurate estimates are crucial to understanding the dynamics of infection and to predicting the effects of different interventions.
 R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of the most important epidemiological parameters [@isbn:9780691116174].
 R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
 A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) compartmental model.
 In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively.
 In this simple formulation, R~0~ is estimated as the ratio of $\beta$ and $\gamma$.[@doi:10.1126/science.abb5659; @isbn:9780691116174].
-A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
+A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996]. 
```

Comment:
@SiminaB says:

> I’m a bit unclear on this. It only takes 1 person to be infected. I know that if Rt < 1 eventually the infectious dies down, but I’m confused on this issue of “invading a susceptible population”.

Reply:
I'm tagging @LucyMcGowan, @alavendelm, and @rdvelazquez for help on this one! I think I understand the idea being communicated but I'm not sure exactly how to adjust the phrasing.